ui/app/services/capabilities.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Service, { service } from '@ember/service';
+
+import type StoreService from 'vault/services/store';
+
+export default class CapabilitiesService extends Service {
```

Comment:
could also call this `@permissions` if we're worried about confusing it with the `@lazyCapabilities` method